src/icrar/leap-accelerate-cli/main.cc

```diff
@@ -51,18 +50,34 @@ namespace icrar
         APACHE_ARROW
     };
 
+    /**
+     * Raw set of command line interface arguments
+     */
     struct Arguments
     {
         InputType source = InputType::FILENAME;
-        boost::optional<std::string> filePath;
-        boost::optional<std::string> directions;
+        boost::optional<std::string> filePath = boost::none; // Measurement set filepath
+        boost::optional<std::string> configPath = boost::none; // Config filepath
+
+        boost::optional<std::string> stations = boost::none;
+        boost::optional<std::string> directions = boost::none;
         ComputeImplementation implementation = ComputeImplementation::casa;
+
+        Arguments() {}
     };
 
+    /**
+     * Validated set of command line arguments
+     */
     class ArgumentsValidated
     {
         InputType m_source;
         boost::optional<std::string> m_filePath;
```

Comment:
This wasn't part of this PR, but I noticed that these two don't really need to be members

Reply:
I haven't tried testing this with multiple file types however my reasoning for expanding on the filepath argument is to allow for things like converting a relative path to an absolute path, handling an ssh path and something like a --verbose flag to print these things to the terminal to help a user diagnose potential issues with the input